Maj m5 dec2021

jengelaere opened the pull request:
Mise à jour du renv.lock
Mise à jour exercice 6 (remplacement par celui de {savoirfR}
Corrections de coquilles
Ajout expand(0,0) dans la partie scale
Relargage de la partie tmap au niveau des ressources

jengelaere committed:
correction typo, repertoire data, et mise à jours crs World,
suprresion renv

jengelaere committed:
MAJ renv

jengelaere committed:
ajour init_M5.sh pour lancement Onyxia

jengelaere committed:
MAJ version packages renv --> fin des problèmes liés aux objets géo
Maj exercice 6 à partir de celui de savoir faire
Corrections typo
Passage du chapitre 'Cartographie avec tmap' historique dans les ressources

jengelaere committed:
update biblio

jengelaere commented:
Désolée, je shunte la branch de dev pour tester la CI toute neuve de Jean Daniel

jdlom commented:
Bizarre elle s'est pas déclenchée. Je la déclenche manuellement

jdlom commented:
ok : c'est pas type `created`mais `opened`

jengelaere committed:
udpate package renv et dernière syncro renv::snapshot

jdlom committed:
fix: renv.lock

jdlom commented:
Il y avait un problème sur la version de waffle en 1.X... je l'ai corrigée.

jdlom committed:
maj wfl rstudio-image

Par rapport au modif sur master

jdlom committed:
add bookdown ci

jdlom commented on 02-présentation-du-package-ggplot2.Rmd:
```diff
@@ -19,23 +19,23 @@ Pour construire un graphique avec ggplot2, il faut définir plusieurs éléments
 - **la donnée** : ggplot2 permet de travailler sur des vecteurs, des dataframes, des tibbles, ou des données spatiales ;
 
-- le **mapping** : on définit dans l'aesthetic (ou aes) le **mapping**, c'est à dire ce que l'on veut représenter qui **dépend des variables** (quelle variable sur l'axe x, sur l'axe y, quelle variable pour définir une graduation de couleurs...) ;
+- le **mapping** : on définit dans l'aesthetic (ou aes) le **mapping**, c'est à dire la correspondance entre éléments visuels et les variables. C'est dans l'aesthetic (ou aes) qu'est déclaré que l’on veut représenter qui **dépend des variables** (quelle variable sur l'axe x, sur l'axe y, quelle variable pour définir une graduation de couleurs...) ;
```
je supprimerai le deuxième aes entre parenthèse

jdlom replied:
je mettrai peut être aesthetic en italique _aesthetic_

jdlom reviewed (commented):
Tout bon pour moi
J'ai fiat 2 remarques sur la partie _aesthetic_ mais c'est vraiment pour t'enquiquiner

jengelaere committed:
coquilles etsuppression waffle

jengelaere committed:
coquilles

jengelaere committed:
Review Murielle

Merge branch 'maj_m5_dec2021' of github.com:MTES-MCT/parcours_r_module_datavisualisation into maj_m5_dec2021

# Conflicts:
#	renv.lock
